vars/signArtifacts.groovy

```diff
@@ -89,56 +90,58 @@ void call(Map args = [:]) {
 
                         echo "------------------------------------------------------------------------"
                         echo "Clean up gpg"
-                        gpg --batch --yes --delete-secret-keys $SIGN_ASM_KEYID
-                        gpg --batch --yes --delete-keys $SIGN_ASM_KEYID
+                        gpg --batch --yes --delete-secret-keys C5B7498965EFD1C2924BA9D539D319879310D3FC
```

Comment:
I don't like we explicit show it here, even though it's public. It may cause other confused.

Reply:
Consulted with @peterzhuamazon. This is a public key as on our website https://opensearch.org/verify-signatures.html. I dont feel like its an env variable or secret as such.